src/main/resources/i18n/messages_ru_RU.properties

```diff
@@ -0,0 +1,2 @@
+settings.chat.notifications=Notifications
+settings.chat.notifyOnAtMentionOnly=Only notifiy when mentioned via @username.
```

Comment:
Don't add such "default" translations; Java will automatically fall back to the default language file if no key is found in the matching language file.